backend/src/src-moderation/src/main/java/com/siemens/sw360/moderation/db/ModerationDatabaseHandler.java

```diff
@@ -280,24 +298,31 @@ public void createRequest(User user) {
         return moderators;
     }
 
-    private Set<String> getAdministrators(String department) {
+    private Set<String> getUsersAtLeast(UserGroup userGroup, String department) {
         List<User> sw360users = getAllSW360Users();
-        List<User> allAdministrators = sw360users
+        List<User> allRelevantUsers = sw360users
                     .stream()
-                    .filter(user1 -> PermissionUtils.isUserAtLeast(UserGroup.ADMIN, user1))
+                    .filter(user1 -> PermissionUtils.isUserAtLeast(userGroup, user1))
                     .collect(Collectors.toList());
-        List<User> administrators = allAdministrators.stream()
+
+        List<User> relevantUsersOfDepartment = Collections.emptyList();
+        if(department != null) {
```

Comment:
True. But we can leave it like this to avoid the work of filtering, when we already know that we do not want to filter.